learning-development/learning-support.qmd

```diff
@@ -48,23 +53,76 @@ In this workshop, we cover:
 
 Contact [statistics.development@education.gov.uk](mailto:statistics.development@education.gov.uk) to register interest in workshops happening at your site or to request new topics!
 
-## Partnership programme
+---
 
- *	Dedicated resource to support development work, either building skills or working on analytical or statistical outputs.
- *	A short-term project with an agreed set of goals and timescale decided as part of an initiation meeting.
- * Contact statistics.development@education.gov.uk if you're interested in finding out more about the support we can give.
+## Partnership programmes
 
-## Specific resources
+The Statistics Development Team offers bespoke partnership programmes for analyst teams across the department, designed as dedicated resource to support development work, either building skills or filling resource gaps to allow improvements in analytical or statistical outputs.
 
-We also have specific learning resources and materials for [SQL](../learning-development/sql.html), [R](../learning-development/r.html), and [Git](../learning-development/git.html)
+Partnerships are designed as short-term projects, agreed in conjunction with the team and their Grade 6. The maximum length for the majority of partnership programmes will be three months, and the project must be clearly defined at the outset.
+
+The aim of partnership programmes is always to upskill the team throughout the project. For this reason, we require any teams taking part to commit time to upskilling during the partnership and being willing to independently continue the work after the partnership is complete, if appropriate. 
```

Comment:
I think you can expand this slightly to highlight the point we would never hand work back to people that can't maintain it. So even if a tier 3, where we do all the work - they'll be expected to put time to follow and upskill alongside to be able to take it on. And then maybe mention the learning path / link back to workshops for what upskilling might look like.

Reply:
I like what you've added and the phrasing of maintaining it independently. The last sentence I might tweak a bit (as ending with signposting to resources feels a bit weak to end on):

```
To enable this, we can offer a variety of support including signposting to resources, recommending relevant L+D, running team workshops and knowledge share sessions, or peer review and pair programming sessions to work side by side building up skills during the partnership.
```

Reply:
Thank you, I hated the last sentence the way I'd written it. Much better now!